modules/moa-accessing-your-cluster.adoc

```diff
@@ -10,31 +10,58 @@ To log in to your cluster, you must configure an Identity Provider (IDP).
-* Open the following URL: https://github.com/organizations/openshift-online/settings/applications/new?oauth_application%5Bcallback_url%5D=https%3A%2F%2Foauth-openshift.apps.rh-moa-test-cluster1.j9n4.s
-1.devshift.org%2Foauth2callback%2Fgithub-1&oauth_application%5Bname%5D=rh-moa-test-cluster1&oauth_application%5Burl%5D=https%3A%2F%2Fconsole-openshift-console.apps.rh-moa-test-cluster1.j9n4.s1.devshift.org   
-* Click on 'Register application'                                                                                                                                                               
-? Copy the Client ID provided by GitHub: <my-github-client-id>                                                                                                                                   
-? Copy the Client Secret provided by GitHub: <my-github-client-secret>                                                                                                           
-I: Configuring IDP for cluster 'rh-moa-test-cluster1'                                                                                                                                                   
-I: Identity Provider 'github-1' has been created. You need to ensure that there is a list of cluster administrators defined. See `moactl user add --help` for more information. To login into th
-e console, open https://console-openshift-console.apps.rh-moa-test-cluster1.j9n4.s1.devshift.org and click on github-1
+I: Interactive mode enabled.
+Any optional fields can be left empty and a default will be selected.
+? Type of identity provider: github
+? Restrict to members of: organizations
+? GitHub organizations: rh-test-org
+? To use GitHub as an identity provider, you must first register the application:
+  - Open the following URL:
+    https://github.com/organizations/rh-moa-test-cluster/settings/applications/new?oauth_application%5Bcallback_url%5D=https%3A%2F%2Foauth-openshift.apps.rh-moa-test-cluster.z7v0.s1.devshift.org%2Foauth2callback%2Fgithub-1&oauth_application%5Bname%5D=rh-moa-test-cluster-stage&oauth_application%5Burl%5D=https%3A%2F%2Fconsole-openshift-console.apps.rh-moa-test-cluster.z7v0.s1.devshift.org
+  - Click on 'Register application'
+...
+----
++
+.. Follow the URL from the output. This will create a new OAuth application in the GitHub organization you specified. 
+.. Click *Register applicaton* to access your Client ID and Client Secret.
+.. Use the information from the GitHub application you just created and continue the prompts. Enter the following values:
++
+* Client ID: <my-github-client-id>
+* Client Secret: [? for help] <my-github-client-secret>
+* Hostname: (optional, you can leave it blank for now)
+* Mapping method: claim
++
+.Continued example output
+[source,terminal]
+----
+...  
+? Client ID: <my-github-client-id>
```

Comment:
`<my-github-client-id>` is rendering as `&lt;my-github-client-id&` in the preview. Same for the below line.